fusor-ember-cli/app/styles/custom.scss

```diff
@@ -594,7 +594,7 @@ table.fusor-table {
 
 .new-deployment-button {
   float: right;
-  margin-top: -33px;
+  margin-top: 24px;
```

Comment:
@cfchase, this fixes the top "New Deployment" button not overlapping with the toolbar. Previously, before `inline-block` it floated right under the <h1>, but now it doesn't.